plugins/qc-s5gen2/qc-s5gen2.quirk

```diff
@@ -1,7 +1,7 @@
 # 5171 devboard connected via USB
 [USB\VID_0A12&PID_4007]
 Plugin = qc_s5gen2
-Flags = enforce-requires
+Flags = no-generic-guids
```

Comment:
I think we want *both* flags, no?

Reply:
No. 
There is no shared GUID for the device in that case, so no need to protect additionally.
The GUID is based on `QC\VID_0A12&PID_4007&MANUFACTURER_Collabora&PRODUCT_CCC5171X`

And some problem: it is inherited somehow, so with enforce flag I have:
```
15:44:25.808 FuMain               requirement on fbd12441fe94a4d805ff32edce5dc1580a71aff2:com.qualcomm.s5gen2.collabora failed: generic GUID requires --force, a CHID, child, parent or sibling requirement
```
the same is without `no-generic-guids`.

**I see that behavior regardless if I add entry into quirk file or not.**

Now it is detected as:
```
└─CCC5171X:
      Device ID:          fbd12441fe94a4d805ff32edce5dc1580a71aff2
      Current version:    1.0.1
      Vendor:             Cambridge Silicon Radio, Ltd (USB:0x0A12)
      Serial Number:      5B23541510FF00B52000
      GUID:               9fe24e67-3d4c-5ed5-8f13-239b6891db9e
      Device Flags:       • Updatable
                          • Device stages updates
                          • Device can recover flash failures
                          • Device is usable for the duration of the update
                          • Signed Payload
                          • Can tag for emulation
```